README.md

```diff
@@ -51,6 +51,7 @@ The [Uxn](https://100r.co/site/uxn.html) ecosystem is a personal computing playg
   - [Raspberry Pi Pico](https://git.sr.ht/~alderwick/pico-uxn) - Partial
   - [Teletype](https://github.com/csboling/teluxn) - Incomplete.
   - [ESP32](https://github.com/max22-/uxn-esp32) - Incomplete.
+  - [iOS](https://github.com/paiv/uxn-ios) - Incomplete.
```

Comment:
Ok, I would say, yes, based on the code here: https://github.com/paiv/uxn-ios/blob/cde481553be8e559a91c6b4d30f3ffb2de9ac8e8/app/Emulator/Platform/app_uxn.c#L163-L178

```
    devsystem = portuxn(u, 0x0, "system", system_talk);
    portuxn(u, 0x1, "console", console_talk);
    devscreen = portuxn(u, 0x2, "screen", screen_talk);
    devaudio0 = portuxn(u, 0x3, "audio0", audio_talk);
    portuxn(u, 0x4, "audio1", audio_talk);
    portuxn(u, 0x5, "audio2", audio_talk);
    portuxn(u, 0x6, "audio3", audio_talk);
    portuxn(u, 0x7, "---", nil_talk);
    portuxn(u, 0x8, "controller", nil_talk);
    devmouse = portuxn(u, 0x9, "mouse", nil_talk);
    portuxn(u, 0xa, "file", file_talk);
    portuxn(u, 0xb, "datetime", datetime_talk);
    portuxn(u, 0xc, "---", nil_talk);
    portuxn(u, 0xd, "---", nil_talk);
    portuxn(u, 0xe, "---", nil_talk);
    portuxn(u, 0xf, "---", nil_talk);
```

That this is incomplete, though more complete than the other iOS project.

The app comes bundled with a lot of ROMs, all of which seems to work, like piano.

Reply:
@neauoire what about the mouse business?